pkg/deployer/clusterapiserver.go

```diff
@@ -73,14 +73,19 @@ func getApiServerCerts() (*caCertParams, error) {
 	return certParams, nil
 }
 
-// GetApiServerYaml returns the clusterapi-apiserver manifest used for deployment
+// GetApiServerYaml returns the clusterapi-apiserver manifest for deployment in the default namespace
```

Comment:
Maybe we should not introduce this function at this point because as far as I see it's not being used.

Reply:
This function was explicitly exported in a previous change. I didn't want to break other folks vendoring this in. IMO introducing this new function is the right change.

Reply:
:+1: Great point!